packages/cosmic-proto/package.json

```diff
@@ -149,14 +153,17 @@
     "@cosmology/telescope": "https://gitpkg.vercel.app/agoric-labs/telescope/packages/telescope?8d2c2f6ba637a5578eead09a7368dc41c262a9d0",
     "@endo/bundle-source": "^3.5.0",
     "@endo/import-bundle": "^1.3.2",
+    "@endo/ses-ava": "^1.2.8",
     "ava": "^5.3.1",
     "rimraf": "^5.0.0",
     "tsd": "^0.31.1",
     "typescript": "~5.7.1"
   },
   "dependencies": {
     "@endo/base64": "^1.0.9",
-    "@endo/init": "^1.1.7"
+    "@endo/init": "^1.1.7",
+    "bech32": "^2.0.0",
+    "query-string": "^9.1.1"
```

Comment:
supply chain audit notes: I don't see any yarn.lock changes re `bech32`, so I'll pass on that without further remark.

`query-string@^9.1.1` seems to be new. npm shows 13M weekly downloads. Clearly pretty mature stuff.